modules/dataframe/src/processors/dataframefilter.cpp

```diff
@@ -76,6 +76,13 @@ DataFrameFilter::DataFrameFilter()
 
     includeFilters_.setCollapsed(true);
     excludeFilters_.setCollapsed(true);
+
+    // make this processor a sink if brushing and linking filtering is selected
```

Comment:
Also, maybe explain why instead?
Make sink because ....
```suggestion
    // make this processor a sink if brushing and linking filtering is enabled
```

Reply:
This is independent of whether Brushing and Linking is enabled, it refers to the dropdown option of "filtering". The reason being that in this case you can use the processor to filter indices without ever using its output.

Reply:
Fixed comment